ang/test/mocks/data/activities.data.js

```diff
@@ -0,0 +1,133 @@
+(function (angular, _) {
+  var module = angular.module('civicase.data');
+
+  module.service('activitiesMockData', ['datesMockData', function (dates) {
```

Comment:
Can we follow the previous pattern of using a constant?

Reply:
Yes agreed. Lets implement this for other mocked datas aswell in future.
cc @swastikpareek